.github/workflows/build.yml

```diff
@@ -337,6 +337,26 @@ jobs:
   #        name: macos-build
   #        path: |
   #          app/build/macos/Build/Products/Release/butterfly.app
+  build-snap:
```

Comment:
I would put the snap into the release assets under deploy

Reply:
No need to change this job, only add this job to the needs in deploy and download this artifact and put it under the release assets

Reply:
Yup, change the workflow as per your need.